analyses/rna-seq-expression-summary-stats/README.md

```diff
@@ -174,12 +174,13 @@ Input:
 - `../../data/gene-expression-rsem-tpm-collapsed.rds`
 - `../../data/histologies.tsv`
 - `../../data/ensg-hugo-rmtl-mapping.tsv`
+- `../independent-samples/results/independent-specimens.rnaseq.primary.tsv`
```

Comment:
```suggestion
- `../../data/results/independent-specimens.rnaseq.primary.tsv`
```
Directory for this and the one after can be changed as well.

Reply:
@runjin326, Sure, I'll update immediately and resubmit the PR